src/main/resources/edu/cornell/kfs/sys/cu-spring-sys.xml

```diff
@@ -255,5 +255,19 @@
 	<bean id="postalCodeValidationService-parentBean" class="org.kuali.kfs.sys.service.impl.PostalCodeValidationServiceImpl" abstract="true">
 		<property name="stateService" ref="stateService" />
     </bean>
-	
+
+	<bean id="documentstoreClient" parent="documentstoreClient-parentBean"/>
+		<bean id="documentstoreClient-parentBean" abstract="true" class="${kfs.documentstore.client.class}">
+			<constructor-arg value="${kfs.documentstore.host}" type="java.lang.String" />
+		    <constructor-arg value="${kfs.documentstore.port}" type="int" />
+		    <constructor-arg value="${kfs.documentstore.dbname}" type="java.lang.String" />
+		    <constructor-arg value="${kfs.documentstore.username}" type="java.lang.String" />
+		    <constructor-arg value="${kfs.documentstore.password}" type="java.lang.String" />
```

Comment:
I don't think these implementations are expected to share anything except the constructor argument lists and we probably only need to enforce constructor initialization. What would be the benefit of replacing the constructor call with setter calls?